### PR TITLE
Feature/events from sheets

### PR DESCRIPTION
**Pull request by kaupunki-apina**
Added a script for fetching events from Google sheets that needs to be set up as a scheduled task on Heroku.

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1,2 @@
 web: NODE_ENV=production ./node_modules/.bin/babel-node --presets es2015 ./src/index.js
+event-worker: node ./src/worker/fetch-events.js
```
Comment (tomi) [inline Procfile]: TIL! Didn't know you can configure workers in Procfile 👍

File: src/worker/fetch-events.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+const _ = require('lodash');
+const google = require('googleapis');
+const util = require('../util/seeds');
+const logger = require('../util/logger')(__filename);
+const {knex} = require('../util/database').connect();
```
Comment (tomi) [inline src/worker/fetch-events.js]: Are we using node 6? If I'm not mistaken we were using node 4 for the backend

File: src/worker/fetch-events.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  'GSHEETS_EVENTS',
+  'GSHEETS_API_KEY',
+]);
+
+const eventSheets = JSON.parse(process.env.GSHEETS_EVENTS);
+const batchGetAsync = BPromise.promisify(sheets.spreadsheets.values.batchGet);
+const cities = {};
+
+getEvents()
+.then(() =>{
```
Comment (tomi) [inline src/worker/fetch-events.js]: Missing space between => and {

File: src/worker/fetch-events.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  process.exit();
+})
+.catch(err => {
+  logger.error("Updating events errored", err);
+  process.exit(1);
+});
+
+function getEvents() {
+  logger.info("Updating events data");
+  return knex('cities').select('*').then(rows => {
```
Comment (tomi) [inline src/worker/fetch-events.js]: Would be better to use `cities-core` for this

File: src/worker/fetch-events.js (hunk 2 above)
Comment (tomi) [inline src/worker/fetch-events.js]: I meant that destructuring is not available in node 4. If I remember correctly we run specifically node v4 in heroku, but I might be mistaken. The fix would be to not use destructuring here or use babel-node

Comment (kaupunki-apina) [inline src/worker/fetch-events.js]: Already made the switch to use transpiled version.

Comment (tomi) [inline src/worker/fetch-events.js]: 👍

**Pull request merged**